docs-src/cloud-context/saml-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+id: saml-intro
+title: How to manage SAML authentication with Temporal Cloud
+sidebar_label: Manage SAML
+description: To authenticate users of your Temporal Cloud account, you can connect a SAML IdP.
+tags:
+  - guide-context
+---
+
+To authenticate the users of your Temporal Cloud account, you can connect a Security Assertion Markup Language (SAML) identity provider (IdP).
+
+If you want to use the general Microsoft login mechanism, you don't need to set up SAML with Azure AD.
+Just select **Continue with Microsoft** on the Temporal Cloud sign-in page.
+
+## Integrate SAML with your Temporal Cloud account
+
+1. Locate your [Temporal Cloud Account Id](/concepts/what-is-a-cloud-account-id).
+   One way to do so is to sign in to Temporal Cloud and find your [Namespace Id](/concepts/what-is-a-cloud-namespace-id).
+   The Account Id is the five or six characters following the period (.), such as `f45a2`.
+   You will need the Account Id to construct your callback URL and your entity identifier.
+1. Configure SAML with your IdP by following one of these sets of instructions:
```

Comment:
Can we also include a generic SAML integration section? In addition to the Azure and Okta samples?

Reply:
Reminder for me: in this generic section we need to ask them to configure their IdP to send the following:
> Please send over the following SAML subject `NameID` and attribute claims:`emailaddress` and `name`  (some IdPs do not configure this automatically)

> 
> 
> ```xml
> <saml:Subject>
> 	<saml:NameID Format="urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"> useremail@example.com </saml:NameID>
>   ...
> </saml:Subject>
> ...
> <saml:AttributeStatement xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
> 	<saml:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
> 		<saml:AttributeValue xsi:type="xs:string"> FirstName LastName </saml:AttributeValue>
> 	</saml:Attribute>
> 	<saml:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
> 		<saml:AttributeValue xsi:type="xs:string"> useremail@example.com </saml:AttributeValue>
> 	</saml:Attribute>
> </saml:AttributeStatement>
> ```
>

Reply:
Sure! I designed this set of docs so that we can easily add IdPs later. All I need is the procedure. (I could draft a generic version from the existing info, but I suspect that it'll need additional context.)